docs/docs/keymaps/behaviors/tap-dance.mdx

```diff
@@ -8,23 +8,75 @@ import TabItem from "@theme/TabItem";
 
 ## Summary
 
-A tap-dance key invokes a different behavior (e.g. `kp`) corresponding to how many times it is pressed. For example, you could configure a tap-dance key that acts as `LSHIFT` if tapped once, or Caps _Lock_ if tapped twice. The expandability of the number of [`bindings`](#bindings) attached to a particular tap-dance is a great way to add more functionality to a single key, especially for keyboards with a limited number of keys. Tap-dances are completely custom, so for every unique tap-dance key,a new tap-dance must be defined in your keymap's `behaviors`.
+A tap-dance key invokes a different behavior (e.g. `kp`) corresponding to how many times it is pressed.
+For example, you could configure a tap-dance key that acts as `LSHIFT` if tapped once, or Caps _Lock_ if tapped twice.
+The expandability of the number of [`bindings`](#bindings) attached to a particular tap-dance is a great way to add more functionality to a single key, especially for keyboards with a limited number of keys.
+Tap-dances accept two parameters from your keymap, which you can map to the behaviors within as you please.
 
 Tap-dances are designed to resolve immediately when interrupted by another keypress. Meaning, when a keybind is pressed other than any active tap-dances, the tap-dance will activate according to the current value of its counter before the interrupting keybind is registered.
 
-### Configuration
+## Keypress Tap Dance
 
-#### `tapping-term-ms`
+ZMK provides a simple two-tap tap dance for you to use. Press it once, and it will output a key press with the first parameter.
+Press it twice within 200ms, and it will output a key press with the second parameter.
+
+### Behavior Binding
+
+- Reference: `&td`
+- Parameters: The keycode usage IDs from the usage page, e.g. `N4` or `A`
```

Comment:
This is a bit more confusing than it should be, but it looks like it is copied from https://zmk.dev/docs/keymaps/behaviors/key-press#behavior-binding. I'd still just call these "keycodes" and link to list of keycodes page.